habitat-hitl/habitat_hitl/_internal/gui_application.py

```diff
@@ -48,30 +49,30 @@ def key_press_event(self, event: Application.KeyEvent) -> None:
                     wrapper._key_down.add(key)
 
     def key_release_event(self, event: Application.KeyEvent) -> None:
-        key = MagnumKeyConverter.convert(event.key)
+        key = MagnumKeyConverter.convert_key(event.key)
         if key:
             for wrapper in self._gui_inputs:
                 if key in wrapper._key_held:
                     wrapper._key_held.remove(key)
                 wrapper._key_up.add(key)
 
     def mouse_press_event(self, event: Application.MouseEvent) -> None:
-        mouse_button = event.button
-        GuiInput.validate_mouse_button(mouse_button)
-        for wrapper in self._gui_inputs:
-            wrapper._mouse_button_held.add(mouse_button)
-            wrapper._mouse_button_down.add(mouse_button)
+        key = MagnumKeyConverter.convert_mouse_key(event.button)
```

Comment:
Maybe still call it mouse button?  Kind of weird to see a 'key' in a mouse event handler. Not a big deal though

Reply:
This makes more sense yeah.